code/validation/adapted-python-files.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""
+23 Jan 2021 - Kath
+Note of files modified for Europe Hub from Germany/Poland Forecast Hub
```

Comment:
It would be great to also adapt the file lists in `decisions.md` according to the things listed here

Reply:
done